core/components/footer/payment-icons/amazon.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 export const AmazonIcon: React.FC<React.ComponentPropsWithoutRef<'svg'>> = ({ ...props }) => (
   <svg
+    aria-hidden="true"
```

Comment:
Wouldn't it be better to use an `aria-label` instead of `aria-hidden`?

Reply:
I intended to remove this from the accessibility API because it's not part of interactive elements. However, I shouldn't be hardcoding aria-hidden into the SVG after looking at this again. It would make more sense to pass this as a prop because there might be a case where this icon should not be hidden.

The icons do need a title, description, and other accessibility attributes. I already have an issue for that, so I think it would make more sense to do that in a separate PR.

Reply:
In the realm of accessibility, it's better to use the `<title>` element inside an `<svg>` element rather than `aria-label`. We could provide a prop, `title` and set `aria-hidden` to `true` if `title` is not provided. We do this in BigDesign: https://github.com/bigcommerce/big-design/blob/main/packages/big-design-icons/src/components/AddIcon.tsx#L10-L11

Reply:
TIL. Thanks @chanceaclark!